apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -108,6 +109,8 @@ const ChatWorkspace: React.FunctionComponent<ChatWorkspaceProps> = ({
     {
       value: 'viewStudentChatHistory',
       text:
+        // TODO: Seems multiple tabs share a hidden "view only" functionality
```

Comment:
Coming back to this comment - there are a bunch of these with the ' view only' suffix in `ModelCustomizationWorkspace.tsx` - I wonder if a string called something like `tabLabelWithSuffix` '{tabLabel} (view only)' would be more appropriate so we aren't tacking 2 translated strings together

Reply:
RTL translations will be more coherent, and we aren't relying on translators leaving that leading space in